index.html

```diff
@@ -387,6 +387,53 @@ <h3>Presentations</h3>
     </section>
 
     <section>
+      <h2>Multiple Times</h2>
+
+      <p>
+Many <a>verifiable credentials</a> will be issued from physical credentials that
+have a much longer validity time than the <a>verifiable credential</a> itself
+e.g. a passport VC, a driving licence VC, a vaccination record VC, a European
+Health Card VC etc. A common requirement might be to reveal this longer validity
+time in the <a>verifiable credential</a>, which leads to 4 times being issued, namely:
+    <p>
+    <ul>
+    <li>
+ the time the original credential was issued or is valid from
+     </li>
+      <li>
+    the time the <a>verifiable credential</a> becomes valid
+     </li>
+      <li>
+    the time the <a>verifiable credential</a> becomes invalid
+     </li>
+      <li>
+    the time the original credential expires
+     </li>
+      </ul>
+  <p>
+It is recommended to use the following properties for the above times:
+   <ul>
+    <li>
+<code>issuedAt</code> - the date/time the original credential was issued
+     </li>
+      <li>
+<code>validFrom</code> - the date/time the <a>verifiable credential</a> becomes valid
+     </li>
+      <li>
+<code>validUntil</code> - the date/time the <a>verifiable credential</a> becomes invalid
+     </li>
+      <li>
+<code>expiresAt</code> - the date/time the original credential expires
+     </li>
+      </ul>
+
+     </p>
+Note. We will need to add issuedAt and expiresAt to the new implementors guide context
+to be published at "https://www.w3.org/2018/credImpGuide/v1" then this note
+can be removed.
```

Comment:
```suggestion
can be removed.</p>
```

Reply:
This PR can be dropped, because the issuer of a VC can publish its own schema for the contents of its VCs and the properties of the VC's subject. If the issuer wants the subject's properties to contain half a dozen different dates and times it can do this without affecting the VC data model or any other VC issuer.